### PR TITLE
Feature/behaviour annotations

### PR DESCRIPTION
**Pull request by VWSCoronaDashboard18**
Added the timeline annotations for the behaviour page

### CODE REVIEW

File: packages/app/src/domain/behavior/behavior-line-chart-tile.tsx [hunk 1]
```diff
@@ -114,7 +120,9 @@ export function BehaviorLineChartTile({
           ]}
           dataOptions={{
             isPercentage: true,
-          }}
+            useDatesAsRange: useDatesAsRange,
+            timelineEvents: timelineEvents,
+          } as DataOptions}
```
Comment (VWSCoronaDashboard24) [inline packages/app/src/domain/behavior/behavior-line-chart-tile.tsx:125]:
```suggestion
          }}
```

File: packages/app/src/domain/behavior/behavior-line-chart-tile.tsx [hunk 2]
```diff
@@ -11,6 +11,8 @@ import { ChartTile } from '~/components/chart-tile';
 import { InlineTooltip } from '~/components/inline-tooltip';
 import { MetadataProps } from '~/components/metadata';
 import { TimeSeriesChart } from '~/components/time-series-chart';
+import { DataOptions } from '~/components/time-series-chart/logic';
```
Comment (VWSCoronaDashboard24) [inline packages/app/src/domain/behavior/behavior-line-chart-tile.tsx:14]:
```suggestion
```

File: packages/app/src/pages/landelijk/gedrag.tsx [hunk 3]
```diff
@@ -108,6 +111,26 @@ export default function BehaviorPage(
...
+      })
+    );
+
+    return { currentTimelineEvents };
+  }, [currentId, data.behavior_annotations.values, siteText.pages.behaviorPage.shared]);
+
+  let timelineProp = {}
+  if (behaviorAnnotationsFeature.isEnabled) {
+    timelineProp = {timelineEvents: currentTimelineEvents,}
+  }
```
Comment (VWSCoronaDashboard24) [inline packages/app/src/pages/landelijk/gedrag.tsx:132]:
```suggestion
  const timelineProp = behaviorAnnotationsFeature.isEnabled
    ? { timelineEvents: currentTimelineEvents }
    : undefined;
```

Comment (VWSCoronaDashboard24) [reply packages/app/src/pages/landelijk/gedrag.tsx:132]: Maybe a bit cleaner. We should prevent mutations where possible

File: packages/app/src/components/time-series-chart/components/axes.tsx [hunk 4]
```diff
@@ -225,31 +311,18 @@ export const Axes = memo(function Axes({
       <AxisBottom
         scale={xScale}
         tickValues={xTicks}
-        tickFormat={formatXAxis as AnyTickFormatter}
+        tickFormat={useDatesAsRange && values 
+          ? ((x: number) => formatXTickValue(x, values as DateSpanValue[])) as AnyTickFormatter
```
[the remark this answers is not in the document]
Comment (VWSCoronaDashboard18) [reply packages/app/src/components/time-series-chart/components/axes.tsx:315]: Screenshot 2022-03-28 at 11 44 33

It seems necessary.

**Review by VWSCoronaDashboard24 — APPROVED** [review]

**Pull request merged**